index.html

```diff
@@ -1913,6 +1914,31 @@ <h5 id="Psionic Traits">Psionic Traits</h5>
                <td>Psionic Level 1
                </td>
             </tr>
+            <tr>
+               <td>Create Vision
+               </td>
+               <td>
+                  You may touch a creature to make them see a vivid vision of your choosing.  This can persist as long as you remain in contact 
+                  with the creature.  They are free to move away and are aware of the source of the vision.
+               </td>
+               <td>1 AP - 3 Uses
+               </td>
+               <td>Psionic Level 3
+               </td>
+            </tr>
+            <tr>
+               <td>Detect Thoughts
+               </td>
+               <td>
+                  You may sense intelligent creatures in a 20ft radius as spots of light when you close your eyes.
+                  Creatures have to have an intelligence of at least 8 for you to be able to see them, and the more intelligent
+                  the creature, the brighter the spots of light.
+               </td>
```

Comment:
Ye cool, could be a bit strong for frequent use seeing through walls, detecting threats in the next room constantly, from a GM perspective it can make it harder to build suspense if they can check through walls constantly. Not exactly sure what we'd do to remedy that, perhaps just a shorter range.